<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IntroForm</class>
 <widget class="QWidget" name="IntroForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <property name="leftMargin">
    <number>80</number>
   </property>
   <property name="topMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>80</number>
   </property>
   <property name="bottomMargin">
    <number>90</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="upperNavigator">
     <property name="spacing">
      <number>200</number>
     </property>
     <property name="leftMargin">
      <number>230</number>
     </property>
     <property name="rightMargin">
      <number>160</number>
     </property>
     <property name="bottomMargin">
      <number>50</number>
     </property>
     <item>
      <widget class="QPushButton" name="muteSwitch">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>120</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="rule">
       <property name="maximumSize">
        <size>
         <width>230</width>
         <height>120</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Microsoft JhengHei</family>
         <pointsize>18</pointsize>
        </font>
       </property>
       <property name="text">
        <string>規則</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="gameTitle">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>160</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Microsoft JhengHei</family>
       <pointsize>30</pointsize>
      </font>
     </property>
     <property name="text">
      <string>游戲標題</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="gamemodeBox">
     <property name="leftMargin">
      <number>200</number>
     </property>
     <property name="rightMargin">
      <number>200</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="chooserAligner">
       <item>
        <widget class="QComboBox" name="chooser">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>70</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>400</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>25</pointsize>
          </font>
         </property>
         <property name="currentText">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="explanation">
       <property name="maximumSize">
        <size>
         <width>900</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Microsoft JhengHei</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>模式介紹</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="startGame">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>130</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Microsoft JhengHei</family>
       <pointsize>27</pointsize>
      </font>
     </property>
     <property name="text">
      <string>開始游戲</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
